libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1852,16 +1918,694 @@ namespace SohImGui {
+                // Blue Potion Mana Percent Restore
+                CVar_SetS32("gBlueManaPercentRestore", 0);
+                // Change Milk Effect
+                CVar_SetS32("gMilkEffect", 0);
+                // Milk Health (1 to 100)
+                CVar_SetS32("gMilkHealth", 1);
+                // Milk Percent Restore
+                CVar_SetS32("gMilkPercentRestore", 0);
+                // Separate Half Milk Effect
+                CVar_SetS32("gSeparateHalfMilkEffect", 0);
+                // Half Milk Health (1 to 100)
+                CVar_SetS32("gHalfMilkHealth", 0);
+                // Half Milk Percent Restore
+                CVar_SetS32("gHalfMilkPercentRestore", 0);
+                // Change Fairy Effect
+                CVar_SetS32("gFairyEffect", 0);
+                // Fairy (1 to 100)
+                CVar_SetS32("gFairyHealth", 1);
+                // Fairy Percent Restore
+                CVar_SetS32("gFairyPercentRestore", 0);
+                // Change Fairy Revive Effect
+                CVar_SetS32("gFairyReviveEffect", 0);
+                // Fairy Revival (1 to 100)
+                CVar_SetS32("gFairyReviveHealth", 1);
+                // Fairy Revive Percent Restore
+                CVar_SetS32("gFairyRevivePercentRestore", 0);
+
+                // Instant Fishing
+                CVar_SetS32("gInstantFishing", 0);
+                // Guarantee Bite
+                CVar_SetS32("gGuaranteeFishingBite", 0);
+                // Child Minimum Weight (6 to 10)
+                CVar_SetS32("gChildMinimumWeightFish", 10);
+                // Adult Minimum Weight (8 to 13)
+                CVar_SetS32("gAdultMinimumWeightFish", 13);
+
+                // Mute Low HP Alarm
+                CVar_SetS32("gLowHpAlarm", 0);
+                // Minimal UI
+                CVar_SetS32("gMinimalUI", 0);
+                // Disable Navi Call Audio
+                CVar_SetS32("gDisableNaviCallAudio", 0);
+
+                // Visual Stone of Agony
+                CVar_SetS32("gVisualAgony", 0);
+                // Assignable Tunics and Boots
+                CVar_SetS32("gAssignableTunicsAndBoots", 1);
+                // Link's Cow in Both Time Periods
+                CVar_SetS32("gCowOfTime", 0);
+                // Enable visible guard vision
+                CVar_SetS32("gGuardVision", 0);
+                // Enable passage of time on file select
+                CVar_SetS32("gTimeFlowFileSelect", 1);
+                // Allow the cursor to be on any slot
+                CVar_SetS32("gPauseAnyCursor", 0);
+                // Count Golden Skulltulas
+                CVar_SetS32("gInjectSkulltulaCount", 1);
+                // Pull grave during the day
+                CVar_SetS32("gDayGravePull", 0);
+
+                // Rotate link (0 to 2)
+                CVar_SetS32("gPauseLiveLinkRotation", 0);
+                // Pause link animation (0 to 16)
+                CVar_SetS32("gPauseLiveLink", 1);
```

Comment:
Up to you if you want to change this, but I added in the latest Zhora build a 17th option to gPauseLiveLink that makes Link's animations cycle like it does when the game isn't paused.

Reply:
I've changed the rando preset to the newest one now. I think for the other presets, the standing link still fits the best. If people want to go change it afterwards, they'll still have the ability to dive into the menus.